dashboard/visualization/plots.py

```diff
@@ -330,14 +329,12 @@ def plot_row_col_means(plate_array: np.ndarray) -> go.Figure:
         ticks = [*range(1, means.shape[0] + 1)]
         ticks_all.append(ticks)
         fig.add_trace(
-            go.Scatter(
+            go.Bar(
                 x=ticks,
                 y=means,
                 error_y=dict(
                     type="data", array=stds, color="gray", thickness=0.5, width=2
                 ),
-                mode="markers",
-                marker_color="blue",
```

Comment:
I think red color is quite misleading here, as it is the same as `control neg`

Reply:
I changed the coloring to more neutral one (that still allows to differentiate between the two)
<img width="321" alt="image" src="https://github.com/zuzg/drug-screening/assets/72276326/9d0e39d3-26ab-4dbb-ad9a-50c541f9cf1a">